libs/blocks/table-of-contents/table-of-contents.css

```diff
@@ -0,0 +1,146 @@
+.section.toc {
+  display: -webkit-box;
+  display: -moz-box;
+  display: -ms-flexbox;
+  display: -webkit-flex;
+  display: flex;
```

Comment:
The other vender prefixes are not needed in 2022.

Reply:
Awesome! Great news!